### PR TITLE
Adding site footer onto Amp pages

### PR DESCRIPTION
**Pull request by NataliaLKB**
image

### CODE REVIEW

File: common/app/views/fragments/footer.scala.html [hunk 1]
```diff
@@ -21,7 +23,7 @@
                     <span class="control__info">back to top</span>
                 </a>
             </div>
-            @if(showNav) {
+            @if(showNav && !amp) {
```
Comment (NataliaLKB) [inline common/app/views/fragments/footer.scala.html]: Not sure when to use `&&` vs `&`.  (cc @johnduffell)

Comment (johnduffell) [inline common/app/views/fragments/footer.scala.html]: ur correct - && is logical and, and & is binary and.  99% of the time you need &&

File: common/app/views/fragments/footer.scala.html [hunk 2]
```diff
@@ -55,7 +57,7 @@
                         daily email sign up</a></li>
                 } else {
                     @if(mvt.EmailTextTestV1.isParticipating) {
-                        @* MVT - Test email text - Variant 1 *@
+                         @* MVT - Test email text - Variant 1 *@
```
Comment (johnduffell) [inline common/app/views/fragments/footer.scala.html]: did you mean to add an indent here?

**Comment by johnduffell** [comment]
I just have a few questions in my mind about what the footer adds... it's a big footer.  should we consider just having one link to a page with the footer on? is downloading it on every page providing enough value to justify it?  Just want to think about other options before we just go ahead and copy the web one.

File: common/app/views/fragments/footer.scala.html (hunk 1 above)
Comment (NataliaLKB) [inline common/app/views/fragments/footer.scala.html]: Okay good to know. Thanks!

File: common/app/views/fragments/footer.scala.html (hunk 2 above)
Comment (NataliaLKB) [inline common/app/views/fragments/footer.scala.html]: Not at all! Will remove.

**Comment by NataliaLKB** [comment]
As far as I am aware, for now we are looking to make AMP pages as close to the site as possible. And although it _is_ a big footer, I do think it is useful. For me, I find the `back to the top` button quite handy. As well as a few of the links. Not all the links are relevant for first time users though. Maybe something smaller like this would be more effective: 

image

**Comment by johnduffell** [comment]
design-wise I'd agree we want to match the main site, but I think we should't just match the features on the main site unthinkingly.  I've always thought the footer is too busy on mobile, so maybe I should be complaining about the main site instead!  I do like your smaller one more, and I agree back to top is useful. Maybe we should chat with @stephanfowler and Jack even.
We could always have an amp-footer dynamic component.

**Comment by crifmulholland** [comment]
I was going to comment too that the footer was big. We need to do some consolidation. We will be moving the email, twitter & fb out soon as this was a short-term benchmarking exercise. It makes sense to look in to the essentials.

**Comment by NataliaLKB** [comment]
@johnduffell @crifmulholland it is too big. For me personally, while reading amp pages there are certain things I look for that I do think we should include - I would include them in _essentials_. They are:
- The `back to the top` button
- The link to `about us`
- The link to `all sections`

About us is especially important because as it stands, if a first time user came to an amp page and wanted to learn more, they wouldn't easily be able to do that.

I think @johnduffell and I will talk to Jack today and find out what links in the footer are actually used on the site more.

**Comment by NataliaLKB** [comment]
@johnduffell and I are still looking at the data, but the changes we will make will be to the full site, not just an amp specific page. Given that we know the footer will change though, here is a temp solution for amp:

image

**Comment by johnduffell** [comment]
looks great! :+1: 
next PR we can add in the about us, contact us, complaints and corrections, and terms and conditions for the amp pages
wondering idly, if we should have a "go to main site" to view the same article - not necessarily in the footer though!

**Pull request merged**